### ISSUE TITLE
Internal Server Error while trying to find miscellaneous infos about some ships

### ISSUE THREAD

**Issue by tommywalkie** [KurozeroPB/al-api#10]
For some ships like _Z23_ or _U-110_ or _Javelin_ for example, it returns a 500 Internal Server Error because there is some trouble when trying to compute miscellaneous data after commit https://github.com/KurozeroPB/al-api/commit/112218944e56b76e47bc4f4f975e62f280f96375 while it still works for some other ships like _Prinz Eugen_ or _Bismarck_.

```json
{
    "statusCode": 500,
    "statusMessage": "Internal Server Error",
    "message": "The server encountered an unexpected condition that prevented it from fulfilling the request.",
    "error": "TypeError: Cannot read property 'attribs' of undefined"
}
```

After some researches, I found why : the voice actress class attribute the script was looking for is not the same for every ships.

_Prinz Eugen_ has `extiw` as a class attribute while _Z23_ has `external text` (_because why not.._) and _U-110_ simply doesn't have VA yet.

**Comment by Pepijn98** [comment]
Ahh shit, I'll fix it later today. Thanks for doing the research makes it a lot easier.

**Comment by Pepijn98** [comment]
Fixed in 4081d87307cdddc9a7eafc18278b4cc0e6539764

**Comment by tommywalkie** [comment]
Nice ! Thanks for the fix ! Have a great weekend ! 👍
